### PR TITLE
Fibonacci Test

### PR DESCRIPTION
**Pull request by csbarrett1**

### CODE REVIEW

**Comment by abidina** [comment]
:+1:

**Comment by csbarrett1** [comment]
@katerebekah

File: Mathmagician.Tests/FibanocciTest.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        public void FibonacciSequenceOfNumbers()
+        {
+            //Arrange
+            Fibonacci my_fibonacci = new Fibonacci();
+            //Act
+            List<int> NumberSequence = my_fibonacci.Sequence(6);
+            List<int> ExpectedSequence = new List<int> { 1, 1, 3, 5, 8, 13 };
+            //Assert
+            Assert.AreEqual(ExpectedSequence.Count, NumberSequence.Count);
+        }
```
Comment (katerebekah) [inline Mathmagician.Tests/FibanocciTest.cs:56]: Add the same CollectionAssert.AreEqual test here for this test.

**Comment by katerebekah** [comment]
Same comments as in Odd

**Comment by katerebekah** [comment]
:+1: (this branch has merge conflicts, so be sure to pull master into your branch and resolve those before merging)

File: Mathmagician/Fibonacci.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician
+{
+    public class FibonacciNum
+    {
+
+        public static int Fibonacci(int n)
```
Comment (katerebekah) [inline Mathmagician/Fibonacci.cs:12]: these methods don't need to be static

**Pull request merged**